test/uk/gov/hmrc/alcoholdutyreturns/repositories/CacheRepositorySpec.scala

```diff
@@ -0,0 +1,83 @@
+/*
```

Comment:
Can you test this in the integration tests as well please? If it is not a big effort